docs/platform/concepts/billing-groups.rst

```diff
@@ -1,35 +1,24 @@
 Billing groups
 ==============
 
-Billing groups enable you to set a common billing profile for multiple
-projects and generate a consolidated invoice.
+Billing groups let you to set up common billing profiles in an organization that can then be used across multiple projects. Instead of entering the payment information every time you create a project, you can use the information saved in the billing group. 
 
-Key benefits of billing groups
-------------------------------
+This makes it easier to manage your costs. You receive a consolidated invoice for all projects assigned to a billing group. This means you can, for instance, combine costs based on your organization or IT environment (development, test, production) by creating billing groups for each of these.
 
-The key benefits of using billing groups are:
+You can also track spending by exporting cost information to business intelligence tools using the `invoice API <https://api.aiven.io/doc/#tag/BillingGroup>`_.
 
--  **Receive a single invoice for all your projects.** When you have
-   several Aiven projects that use the same payment method, you can
-   receive a single invoice that consolidates all costs instead of
-   receiving a separate invoice for each project.
+Types of billing groups
+------------------------
 
--  **Group costs based on your organization need.** You can merge costs
-   based on your organization or IT environment (development, test,
-   production) by creating billing groups for related projects. Each
-   billing group then generates a consolidated invoice.
+There are two types of billing groups:
+* Organization billing groups
+* Project billing groups
 
--  **Minimize administrative setup of billing profiles shared between
-   projects.** Move projects that use the same payment card into a
-   billing group and designate a single billing profile to pay their
-   costs.
+An organization billing group is assigned directly to an organization and can be used across all projects within that organization. This includes projects in organizational units under that organization. Organization billing groups can only be used in one organization. This means that you will not be able to use that billing group for projects that are in other organizations. Aiven credits are available in an organization's billing group.
```

Comment:
I think the "organization billing group" can be changed to just "billing group", since there is no need to talk about "project billing groups" so there is no need to separate the 'organization billing group' either.

Billing groups belong to a an organization and are used to invoice projects on same invoice. This includes projects in any organizational unit under that organization. Billing groups can only be used by projects in the same organization. The aiven credits are assigned per billing group. Any credits in a billing group are automatically used to cover charges from any of the projects assigned to that billing group.